### DISCUSSION TITLE
Anchor part calculator

### DISCUSSION THREAD

**Discussion by roomrys** [talmolab/sleap#1544] [Ideas]
The anchor part can be confusing. Which node should be chosen as the anchor part? A node near the center makes sense, but since the centroid model relies on finding the anchor point, wouldn't a node which is (1) visible more frequently than other nodes (taken from the labeled data) and, for tie-breakers, (2) closest to the centroid of the bounding box make most sense? 

Of course, we can add instructions on this (which there probably already are), or we can add a function that does this. It would be a bit expensive to calculate this using all labeled instances, especially in bigger projects, so we would want to only calculate this on new projects (and/or use a randomly sampled subset of labels to calculate). I imagine this function could run when clicking a "Calculate" button in the Training Dialog or automatically if no previous training configuration is found (making the assumption that the project is newly started -> very bad assumption on bigger projects unless we use a sampled subset).

**Comment by gitttt-1234** [comment]
The [Config Picker](https://sleap-nn.readthedocs.io/en/latest/configuration/config-picker/) in sleap-nn now shows per-node **visibility percentages** alongside an interactive skeleton preview. You can click a node to select it as the anchor, and the visibility stats help you pick the most reliably visible one.
<img width="1199" height="659" alt="image" src="https://github.com/user-attachments/assets/a721265e-5d99-44d3-9369-e08346a3d98c" />


Closing this as addressed!
